tests/TestCase.php

```diff
@@ -93,19 +122,15 @@ protected function clearSolrIndex() {
     protected function setUp() {
         parent::setUp();
 
+        Opus_Search_Config::dropCached();
+        Opus_Search_Service::dropCached();
+
         $this->_clearTables();
         $this->clearSolrIndex();
-
-        $config = Zend_Registry::get('Zend_Config');
-        if (!is_null($config)) {
-            $this->config_backup = clone $config;
-        }
     }
 
-    protected function  tearDown() {
-        if (!is_null($this->config_backup)) {
-            Zend_Registry::set('Zend_Config', $this->config_backup);
-        }
+    protected function tearDown() {
+        $this->clearSolrIndex();
```

Comment:
Actually, I was changing TestCase.php due to doing too much as a basis for some tests. That's why I introduced SimpleTestCase doing now what was removed here. I've chosen to refactor this way to keep impact on change as little as possible for quite a lot of tests in framework are derived from TestCase ... and I needed a single test to derive from what is now provided by SimpleTestCase.